app/screens/thread/thread_post_list/thread_post_list.tsx

```diff
@@ -42,21 +45,29 @@ const ThreadPostList = ({
 }: Props) => {
     const isTablet = useIsTablet();
     const serverUrl = useServerUrl();
+    const theme = useTheme();
+    const fetchThreadState = useFetchThreadState();
 
-    const canLoadPosts = useRef(true);
+    const canLoadMorePosts = useRef(true);
     const fetchingPosts = useRef(false);
```

Comment:
fetchingPosts should be true if fetchThread for the post is true, in fact do you need both?

Reply:
True, we can reuse the new hook for this condition as well.